rerun_js/web-viewer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rerun-io/web-viewer",
-  "version": "0.12.0",
+  "version": "0.13.0-rc.4",
```

Comment:
```suggestion
  "version": "0.13.0",
```

Reply:
Or maybe no, because the npm release didn't go through? @jprochazk

Reply:
Yes, these versions should not be updated right now, I'm going to do the NPM release manually later today.